pkg/server/apidev/init.go

```diff
@@ -43,12 +44,20 @@ func InitTaskHandler(ctx context.Context, s *state.State, r *http.Request, req I
 		return InitTaskResponse{}, libhttp.NewErrBadRequest("missing kind")
 	}
 
+	// Check for slug uniqueness
+	if ok, err := IsTaskSlugAvailable(ctx, s, req.Slug); err != nil {
+		return InitTaskResponse{}, errors.Wrap(err, "checking if slug is available")
+	} else if !ok {
+		return InitTaskResponse{}, libhttp.NewErrBadRequest("slug taken")
```

Comment:
Let's provide more user-friendly error messages. Same for Views, below.

```suggestion
		return InitTaskResponse{}, libhttp.NewErrBadRequest("This slug is already used by another task. Please choose  a unique slug.")
```